packages/react/src/Image/Image.tsx

```diff
@@ -4,12 +4,16 @@
  */
 
 import clsx from 'clsx'
-import { ForwardedRef, forwardRef, ImgHTMLAttributes } from 'react'
+import { type ForwardedRef, forwardRef, type ImgHTMLAttributes } from 'react'
 
-export interface ImageProps extends ImgHTMLAttributes<HTMLImageElement> {}
+export interface ImageProps extends ImgHTMLAttributes<HTMLImageElement> {
+  objectFit?: 'cover' | 'contain'
+}
 
-export const Image = forwardRef(({ className, ...restProps }: ImageProps, ref: ForwardedRef<HTMLImageElement>) => (
-  <img {...restProps} ref={ref} className={clsx('amsterdam-image', className)} />
-))
+export const Image = forwardRef(
+  ({ className, objectFit, ...restProps }: ImageProps, ref: ForwardedRef<HTMLImageElement>) => (
+    <img {...restProps} ref={ref} className={clsx('amsterdam-image', className)} style={{ objectFit }} />
```

Comment:
Using inline styles deviates from the pattern we use in the rest of the codebase. Any reason to not use a BEM class here?

Reply:
I was actually thinking of what you sometimes say about not wanting to replicate CSS :)

Using classes here would introduce a couple of `.amsterdam-image-object-fit--cover { object-fit: cover; }` – would that be better?

Reply:
Yeah, it feels a bit unnecessary, but it makes sense in the system we have imo. Inline styles have some downsides, they aren't cached for example.

Do we ever need `contain`? I'd expect that to mess up the perceived vertical spacing, it would visually add transparent bars to an image.

Reply:
I’ve changed this to a Boolean `cover` prop.